src/arviz_plots/plots/utils.py

```diff
@@ -238,16 +243,21 @@ def add_reference_lines(
         ref_kwargs.setdefault("color", "gray")
     if "linestyle" not in ref_aes:
         ref_kwargs.setdefault("linestyle", plot_bknd.get_default_aes("linestyle", 2, {})[1])
-
+    ref_dt, artist_dims = {}, {}
```

Comment:
very minor comment but can this variable be renamed to `ref_dict`? We generally use `dt` for DataTree, `ds` for Dataset, `da` DataArray and following always the same convention helps read and understand the code faster. I can add a page to the contributing section about this if it helps too